### ISSUE TITLE
Something wrong in my profile page!

### ISSUE THREAD

**Issue by SallamAltanna** [IsraaMoqbel/memehub#4]
Why when I go to the profile page then refresh it I got an error message!

**Comment by IsraaMoqbel** [comment]
Yeah, I just found out this problem! it only appears in production! I don't know what the problem is maybe [netlify](https://www.netlify.com/) has a hand in this!:confused:
